config/externalnamenottested.go

```diff
@@ -382,15 +466,80 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
 	"google_storage_hmac_key": config.IdentifierFromProvider,
 	// Imported by using the following default_bucket/notificationConfigs/102
 	"google_storage_notification": config.IdentifierFromProvider,
+	// Imported by using the following {{bucket}}/{{object}}/{{entity}}
+	"google_storage_object_access_control": config.TemplatedStringAsIdentifier("entity", "{{ .parameters.bucket }}/{{ .parameters.object }}/{{ .external_name }}"),
+	// No Import
+	"google_storage_object_acl": config.IdentifierFromProvider,
+	// Imported by using the following projects/{{project}}/agentPools/{{name}}
+	"google_storage_transfer_agent_pool": config.TemplatedStringAsIdentifier("name", "projects/{{ .setup.configuration.project }}/agentPools/{{ .external_name }}"),
 	// Imported by using the following google_storage_transfer_job.nightly-backup-transfer-job my-project-1asd32/8422144862922355674
 	"google_storage_transfer_job": config.IdentifierFromProvider,
 
 	// vertexai
 	//
+	// No Import
+	"google_vertex_ai_dataset": config.IdentifierFromProvider,
 	// Imported by using the following projects/{{project}}/locations/{{location}}/endpoints/{{name}}
 	"google_vertex_ai_endpoint": config.TemplatedStringAsIdentifier("name", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.location }}/endpoints/{{ .external_name }}"),
+	// Imported by using the following projects/{{project}}/locations/{{region}}/featurestores/{{name}}
+	"google_vertex_ai_featurestore": config.IdentifierFromProvider,
+	// Imported by using the following {{featurestore}}/entityTypes/{{name}}
+	"google_vertex_ai_featurestore_entitytype": config.IdentifierFromProvider,
 	// Imported by using the following {{entitytype}}/features/{{name}}
 	"google_vertex_ai_featurestore_entitytype_feature": config.IdentifierFromProvider,
 	// Imported by using the following projects/{{project}}/locations/{{region}}/indexes/{{name}}
 	"google_vertex_ai_index": config.TemplatedStringAsIdentifier("name", "projects/{{ .setup.configuration.project }}/locations/{{.parameters.region }}/indexes/{{ .external_name }}"),
+	// Imported by using the following projects/{{project}}/locations/{{region}}/tensorboards/{{name}}
+	"google_vertex_ai_tensorboard": config.TemplatedStringAsIdentifier("display_name", "projects/{{ .setup.configuration.project }}/locations/{{ .parameters.region }}/tensorboards/{{ .external_name }}"),
```

Comment:
We may consider marking the `region` field as required.

Reply:
Already marked in config.go